source/framework/core/src/TRestEveEventViewer.cxx

```diff
@@ -22,6 +24,18 @@ TRestEveEventViewer::TRestEveEventViewer() {
     Initialize();
     fEnergyDeposits = new TEvePointSet();
     fEnergyDeposits->SetElementName("Energy deposits");
+
+    RESTWarning << "There are some issues with geometry representation in Eve 3D scenes!" << RESTendl;
+    RESTWarning
+        << "We use a geometry scaling factor to place the hits in the scene and correct a placement problem"
+        << RESTendl;
+    RESTWarning << "Presently the default value of fGeomScale is " << fGeomScale << RESTendl;
+    RESTWarning << "Please, report to the dev-team if you experience problems visualizing the geometry."
```

Comment:
```suggestion
    RESTWarning << "Please, report to the dev team if you experience problems visualizing the geometry."
```

Reply:
I am not 100% sure about this one but I think this would be the preferred form, I may be wrong though.